test/Microsoft.ML.Core.Tests/Microsoft.ML.Core.Tests.csproj

```diff
@@ -22,6 +22,7 @@
     <ProjectReference Include="..\..\src\Microsoft.ML.Legacy\Microsoft.ML.Legacy.csproj" />
     <ProjectReference Include="..\Microsoft.ML.TestFramework\Microsoft.ML.TestFramework.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Tests\Microsoft.ML.Tests.csproj" />
+    <PackageReference Include="LightGBM" Version="$(LightGBMPackageVersion)" />
```

Comment:
><PackageReference Include="LightGBM" Version="$(LightGBMPackageVersion)" /> [](start = 4, length = 75)

Don't we pull this dependency through ProjectReference to LightGBM ?

Reply:
You're right, I see that we do.  I'll remove the unnecessary PackageReferences.